### PR TITLE
Fix indentation

### PR DESCRIPTION
**Pull request by reyang**

### CODE REVIEW

**Review by cijothomas — APPROVED** [review]

**Comment by cijothomas** [comment]
@reyang There is a markdownlint failure to be fixed before merge

File: .editorconfig [hunk 1]
```diff
@@ -4,13 +4,11 @@
 ###############################
 # All files
 [*]
+charset = utf-8
 indent_style = space
+indent_size = 4
```
Comment (alanwest) [inline .editorconfig:9]: We've got a few `yml`, `xml`, and `json` files that are normally `indent_size = 2`. I think this will affect those.

Comment (reyang) [reply .editorconfig]: Good catch, fixed.

File: build/sanitycheck.py [hunk 2]
```diff
@@ -62,23 +72,23 @@ def sanitycheck(pattern, allow_utf8 = False, allow_eol = (CRLF, LF)):
...
+retval += sanitycheck('.editorconfig', allow_eol = (LF,), indent = 0)
+retval += sanitycheck('**/Dockerfile', allow_eol = (LF,), indent = 2)
+retval += sanitycheck('**/*.cmd', allow_eol = (CRLF,), indent = 2)
+retval += sanitycheck('**/*.config', allow_utf8 = True, allow_eol = (LF,), indent = 2)
 retval += sanitycheck('**/*.cs', allow_utf8 = True, allow_eol = (LF,))
-retval += sanitycheck('**/*.cshtml', allow_utf8 = True, allow_eol = (LF,))
-retval += sanitycheck('**/*.csproj', allow_utf8 = True, allow_eol = (LF,))
-retval += sanitycheck('**/*.htm', allow_eol = (LF,))
-retval += sanitycheck('**/*.html', allow_eol = (LF,))
+retval += sanitycheck('**/*.cshtml', allow_utf8 = True, allow_eol = (LF,), indent = 4)
```
Comment (mic-max) [inline build/sanitycheck.py:80]: Will `indent = 4` be the default taken from `.editorconfig`?

**Pull request merged**